.github/workflows/lint.yaml

```diff
@@ -9,13 +13,24 @@ permissions:
   pull-requests: read
 
 jobs:
-  lint:
+  trunk-check:
     runs-on: ubuntu-latest
     steps:
       - name: Check out Git repository
         uses: actions/checkout@v4
       - name: Trunk Check
         uses: trunk-io/trunk-action@v1
+
+  tf-fmt:
```

Comment:
We don't need fmt check if we enable tofu linter via trunk.

Reply:
Great point 👍 Updating.

Reply:
Updated!